azure-functions-codegen/src/util.rs

```diff
@@ -35,16 +43,80 @@ pub struct AttributeArguments {
 }
 
 impl AttributeArguments {
-    pub fn with_name(name: &str, span: ::proc_macro2::Span) -> Self {
+    pub fn with_name(name: &str, span: proc_macro2::Span) -> Self {
         AttributeArguments {
             span: span.unstable(),
             list: vec![(Ident::new("name", span), Lit::Str(LitStr::new(name, span)))],
         }
     }
 }
 
+cfg_if::cfg_if! {
+    if #[cfg(feature = "unstable")] {
+        pub struct MacroError {
+            inner: Diagnostic,
+        }
+        impl MacroError {
+            pub fn emit(self) {
+                println!("A");
```

Comment:
Debug `println!` to remove?

Reply:
Yep! Sorry. I blame all this on my jet lag 😄